projects/Mallard/src/hooks/use-image-paths.ts

```diff
@@ -62,19 +71,17 @@ const useImagePath = (image?: Image) => {
     }, [
         apiUrl,
         image,
-        key ? key.publishedIssueId : undefined,
+        key ? key.publishedIssueId : undefined, // Why isn't this just key?
```

Comment:
Why?

Reply:
it upsets the linter, so I think we could do with a comment here